src/hooks/useExternalLinks.ts

```diff
@@ -0,0 +1,28 @@
+import { ExternalLinks } from '~gql';
+import useColonyContext from './useColonyContext';
+
+export const useExternalLinks = () => {
```

Comment:
Please add a return type here

Reply:
It's also not something that will be used globally, so I think that you can move it to the directory where it's used